tsconfig.esm.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "target": "ES2022",                               
+    "module": "ESNext",                             
+    "declaration": true,           
+    "outDir": "./dist/esm",                            
```

Comment:
```suggestion
    "outDir": "./esm",                            
```

Reply:
updated in: [3ae03a2](https://github.com/KILTprotocol/kilt-extension-api/pull/27/commits/3ae03a28dfb736af8ff82c330a70622277a2bdfe)